release-notes/1-62-0-release-notes.md

```diff
@@ -0,0 +1,58 @@
+# Quilt Platform Release 1.62.0
+
+This release expands support for the QuiltSync desktop sync client and the Qurator AI chatbot,
+and addresses a stack name limitation impacting the prior release.  There is also a new version of `quilt3`
+that streamlines pushing packages across buckets.
+
+## Catalog Enhancements
+
+### HTTP Redirects for Opening QuiltSync
+
+To support clients that cannot directly open the Quilt+ URIs used by [QuiltSync](https://www.quilt.bio/quiltsync),
+we have added a [new `redir` route](https://docs.quilt.bio/quilt-platform-catalog-user/uri#catalog-usage) to the Quilt Catalog.
+Appending the URL-encoded `quilt+s3://` URI for a package or path to `https://your-catalog-host/redir/`
+generates a standard URL that will redirect to the Quilt+ URI, automatically opening QuiltSync (if installed).
+
+For example: `https://open.quiltdata.com/redir/quilt%2Bs3%3A%2F%2Fquilt-example%23package%3Dakarve%2Fcord19%26path%3DCORD19.ipynb`
+
+### Expanded Qurator Developer Tools
+
+The built-in Developer Tools (available in the upper right menu of the Qurator AI chat window) have been expanded with two new features:
+
+![Qurator Developer Tools Screenshot](./1-62-0-qurator-tools.png)
+
+#### Session Recordings
+
+Similar to web inspectors, users can record a portion of their Qurator session,
+and then download (or clear) the resulting JSON log.
+This is primarily intended for tuning or debugging prompts,
+but is also a convenient way to capture structured results.
+
+#### Swappable Models
```

Comment:
I think order does matter, and I'd put this section first, and "HTTP Redirects ..." before "...Stack Names"

Reply:
I mean, from my perspective, the only user-faced feature is "Swappable Models", so it should go first.

Then related and probably used more than other new features: "Qurator Dev Tools" + "Qurater recordings".

Then feature with very narrow case (they should use QuiltSync, need some integration and willing to programmatically change URLs) - "HTTP Redirect and QuiltSync"

Then bugfix/limit feature: "Stack Names"